src/hotspot/share/utilities/growableArray.hpp

```diff
@@ -153,17 +153,31 @@ class GrowableArrayView : public GrowableArrayBase {
     return &_data[i];
```

Comment:
(GitHub won't let me put comment on the `adr_at` signature.)

I think there should similarly be const and non-const adr_at, returning pointer to const and non-const respectively.

Reply:
Done!

Reply:
Alright,

I'm reverting this change. The issue is that C1 and C2 aren't very `const`-correct, and fixing this would make the size of the PR blow up.

Reply:
OK.  Can you file a followup issue to work on that?